forge-gui/res/cardsfolder/upcoming/nalfeshnee.txt

```diff
@@ -0,0 +1,8 @@
+Name:Nalfeshnee
+ManaCost:5 R
+Types:Creature Beast Demon
+PT:4/6
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.wasCastFromExile | ValidActivatingPlayer$ You | Execute$ TrigCopy | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a spell from exile, copy it. You may choose new targets for the copy. If it's a permanent spell, the copy gains haste and "At the beginning of the end step, sacrifice this permanent." (A copy of a permanent spell becomes a token.)
+SVar:TrigCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility | Amount$ 1 | MayChooseTarget$ True | UnearthEffect $ True
+Oracle:Flying\nWhenever you cast a spell from exile, copy it. You may choose new targets for the copy. If it's a permanent spell, the copy gains haste and "At the beginning of the end step, sacrifice this permanent." (A copy of a permanent spell becomes a token.)
```

Comment:
I think this one also needs `AI:RemoveDeck:Random` until we get better AI support for the "I like to cast things from exile" archetype